app/controllers/investigations/ownership_controller.rb

```diff
@@ -79,10 +79,18 @@ def form
   end
 
   def get_potential_assignees
-    @selectable_users = [@investigation.owner_user, current_user].uniq.compact
     @team_members = current_user.team.users.active.includes(:team)
-    @selectable_teams = [current_user.team, Team.get_visible_teams(current_user), @investigation.owner_team].flatten.uniq.compact
     @other_teams = Team.not_deleted
     @other_users = User.active.includes(:team)
+    @other_teams_added_to_case = Team.find(@investigation.collaboration_accesses.where("type IN (?)", ["Collaboration::Access::Edit", "Collaboration::Access::ReadOnly"]).map(&:collaborator_id))
```

Comment:
You could consider using the `changeable` scope here rather than hardcoding class names.
```suggestion
    @other_teams_added_to_case = Team.find(@investigation.collaboration_accesses.changeable).map(&:collaborator_id))
```
Or perhaps look at the `Investigation#teams_with_edit_access` and `Investigation#teams_with_read_only_access` associations and make another combined one.

Reply:
@KyleMacPherson I've now merged into the prototypes master changes to the CSS as highlighted by @slorek . 

This has removed a new class that was overriding a govuk class. The HTML/sass has been updated on page: 
case-owner-change-page/improve-case-owner-change-page

<div ="govuk-radios__divider **opss-width-100**">
 
Please add class .opss-width-100 to the .govuk-radios__divider DIVs and get the latest sass from the opss-psd sass file.